src/main/proto/query.proto

```diff
@@ -3,113 +3,268 @@ package io.axoniq.axonserver.grpc.query;
     map<string, MetaDataValue> meta_data = 5;
+
+    /* An object describing the expectations of the Response Type */
     SerializedObject response_type = 6;
+
+    /* Any instructions for components Routing or Handling the Query */
     repeated ProcessingInstruction processing_instructions = 7;
+
+    /* The unique identifier of the client instance dispatching the query */
     string client_id = 8;
+
+    /* The Name of the Component dispatching the query */
     string component_name = 9;
 }
 
+/* Message that represents the Response to a Query */
 message QueryResponse {
-    /* A message id */
+    /* The unique identifier of the Response Message */
     string message_identifier = 1;
+
+    /* An Error Code identifying the type of error, if any */
     string error_code = 2;
+
+    /* A detailed description of the error, if any */
     ErrorMessage error_message = 3;
+
+    /* The Payload of the Response Message */
     SerializedObject payload = 4;
+
+    /* Any Meta Data describing the context of the Response Message */
     map<string, MetaDataValue> meta_data = 5;
+
+    /* Any instructions for components Routing or Handling the Response Message */
     repeated ProcessingInstruction processing_instructions = 6;
+
+    /* The unique identifier of the Query to which this is a response */
     string request_identifier = 7;
 }
 
+/* Message that represents a Subscription Query */
 message SubscriptionQuery {
+
+    /* A unique identifier for this subscription */
     string subscription_identifier = 1;
+
+    /* The number of messages the Server may send before needing to await additional permits */
     int64 number_of_permits = 2;
+
+    /* The Query describing the desire for information */
     QueryRequest query_request = 3;
+
+    /* A description of the type of Object expected as Update Responses */
     SerializedObject update_response_type = 4;
 }
 
+/* A message containing an Update of a Query Subscription Response */
 message QueryUpdate {
+
+    /* The unique identifier of this Update */
     string message_identifier = 2;
+
+    /* The object representing the Update */
     SerializedObject payload = 3;
+
+    /* Meta Data providing contextual information of the Update */
     map<string, MetaDataValue> meta_data = 4;
+
+    /* The identifier of the Client instance providing the Update */
     string client_id = 5;
+
+    /* The Component Name of the Client providing the Update */
     string component_name = 6;
 }
 
+/* Message indicating that all relevant Updates have been sent for a Subscription Query, and that no further Updates are available */
 message QueryUpdateComplete {
+
+    /* The identifier of the Client instance providing the Update */
     string client_id = 2;
+
+    /* The Component Name of the Client providing the Update */
     string component_name = 3;
 }
 
+/* Message indicating that an Error occurred and that no Updates will sent for a Subscription Query */
```

Comment:
Typo: `...no Updates will be sent for a...` i.o. `...no Updates will sent for a...`